velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -466,6 +470,32 @@ struct TimestampMinusIntervalDayTime {
   }
 };
 
+template <typename T>
+struct TimestampAtTimezoneFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<TimestampWithTimezone>& result,
+      const arg_type<Timestamp>& ts,
+      const arg_type<Varchar>& timeZone) {
+
+    auto timeZoneStr = std::string_view(timeZone.data(), timeZone.size());
+    auto zone = date::locate_zone(timeZoneStr);
+    std::chrono::system_clock::time_point tp{std::chrono::seconds{ts.getSeconds()}};
+
+    // Africa/Abidjan has offset +00:00 -> use this timezone to establish UTC as starting offset
+    auto startZonedTime = date::make_zoned("Africa/Abidjan", tp);
+
+    // find corresponding time at input timezone
```

Comment:
please use full sentences for comments (capitalize and period at the end)

Reply:
Fixed.